.bazelversion

```diff
@@ -0,0 +1 @@
+6.4.0
```

Comment:
For my understanding why did you choose this version? Locally/internally I have been using version 7.0.1 actually - any reason not to use a more up-to-date bazel version?